parent/pom.xml

```diff
@@ -85,6 +85,7 @@
     <version.restassert>4.3.0</version.restassert>
     <version.spring-framework>5.2.7.RELEASE</version.spring-framework>
     <version.spring-boot>2.3.1.RELEASE</version.spring-boot>
+    <version.tomcat>9.0.31</version.tomcat>
```

Comment:
From the posted CVE, the affected versions span from `9.0.0` (including) to `9.0.35` (excluding). Did you mean to pin this to `9.0.35`?

Reply:
Oops, I took it from the second CVE they link to where it says fixed in 9.0.31 :s